app/views/notifications/_aggregated_reactions.html.erb

```diff
@@ -40,7 +40,10 @@
           reaction_categories.filter_map do |cat|
             image_path = reaction_image(cat)
             if image_path.present?
-              crayons_icon_tag(image_path, class: "reaction-image mx-1 my-1 reaction-icon--#{cat}", title: t("views.reactions.category.#{cat}"))
+              "<span class='crayons-tooltip__activator' style='position: relative;'>
+                #{crayons_icon_tag(image_path, class: "reaction-image mx-1 my-1 reaction-icon--#{cat}")}
+              <span class='crayons-tooltip__content'>#{t("views.reactions.category.#{cat}")}</span>
+              </span>".html_safe
```

Comment:
```suggestion
            "<span class='crayons-hover-tooltip inline-block' data-tooltip='#{t("views.reactions.category.#{cat}")}'>
                #{crayons_icon_tag(image_path, class: "reaction-image mx-1 my-1 reaction-icon--#{cat}")}
              </span>".html_safe
```

The "normal" tooltip is designed to be used only within buttons (for accessibility reasons), but we have a class for these special cases where we want one outside of the button context. It saves us adding the extra span 🙂